### PR TITLE
Enable flake8 rule E225 which checks for missing whitespace around op…

### PR DESCRIPTION
**Pull request by jbampton**
…erators

Lint Python code for rule E225

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -116,3 +116,5 @@ test/cache/*
 
 libbitcoinconsensus.pc
 contrib/devtools/split-debug.sh
+
+.idea
```
Comment (practicalswift) [inline .gitignore]: Remove this. [See "Ignoring IDE/editor files" in the developer notes](https://github.com/bitcoin/bitcoin/blob/f504a1402afd0760e9d348ecc8bad0094aa7d705/doc/developer-notes.md#ignoring-ideeditor-files) :-)

**Comment by practicalswift** [comment]
ACK 74f4e2d648eb40dd2315b46d87d6824f9f419e56 assuming nit is addressed and squash :-)

**Comment by practicalswift** [comment]
ACK 0ad199b543465f6aead05ee9014907eca36049e5

Nice to never have to think or comment about this on this in future code reviews thanks to this check: review automation FTW :-)

**Comment by Empact** [comment]
utACK 0ad199b changes are whitespace only (https://github.com/bitcoin/bitcoin/pull/14534/files?w=1), the result is easier to read and the practice is automatically enforced

**Comment by jbampton** [comment]
Hey everyone, I did use autopep8 for this. 

From the project root directory the command I ran was:

```
autopep8 --in-place --select=E225 -r .
```

I would be happy to add other flake8 checks if you want to this pull request.

autopep8 uses pycodestyle

https://pep8.readthedocs.io/en/latest/intro.html#error-codes

Here is the link to autopep8:

 https://github.com/hhatto/autopep8

**Comment by ken2812221** [comment]
What I mean is writing the commands you were using into the commit message by [scripted-diff](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#scripted-diffs). Then travis-ci will run the commands again to check if it is the same. So reviewers won't have to run the commands manually on their own computer to check if the change is correct or not.

For example (Haven't tested): 
```
-BEGIN VERIFY SCRIPT-
pip3 install --user autopep8
autopep8 --in-place --select=E225 -r .
-END VERIFY SCRIPT-

**Comment by jbampton** [comment]
Hey @ken2812221 thanks for explaining the process further.  

Does that mean you want me to commit again with a scripted diff ?

**Comment by promag** [comment]
How can we ensure other pulls don't break master if merged after this?

+1 scripted diff.

**Comment by practicalswift** [comment]
@promag I usually nit-comment on this and I believe I've reviewed all open mergeable (non-conflict) PRs touching Python code.

FWIW the following open mergeable PRs touch Python code: #14539, #14534, #14531, #14530, #14522, #14519, #14518, #14507, #14504, #14494, #14489, #14481, #14476, #14468, #14459, #14457, #14454, #14453, #14433, #14430, #14417, #14416, #14411, #14410, #14391, #14380, #14365, #14356, #14353, #14350, #14320, #14296, #14197, #14194, #14150, #14121, #14092, #14075, #14066, #14060, #14021, #13998, #13972, #13932, #13926, #13746, #13728, #13693, #13541, #13472, #13381, #13357, #13315, #12911, #12656, #12419, #12134, #12096, #11413, #10595, #10443 and #8994.

**Comment by ken2812221** [comment]
@jbampton You can `git commit --amend` to override the commit message and force push it.

@promag Yes, I assume travis does not have `autopep8` installed. Also we don't want to change travis script. I think this is the only way to install it.

**Comment by MarcoFalke** [comment]
Both variants are allowed in python and the whitespace is not really "missing". Tend to NACK, as I don't see the advantage of doing this code churn for everyone.

**Comment by MarcoFalke** [comment]
See also #14540

**Comment by MarcoFalke** [comment]
Trivial pull request still needs rebase. Closing for now.

**Pull request closed without merging**